src/components/common/ColonyHome/types.ts

```diff
@@ -0,0 +1,22 @@
+import { BigNumber } from 'ethers';
+import { TokenType } from '~gql';
+import { UserAvatarsItem } from '~v5/shared/UserAvatars/types';
+
+export interface UseGetHomeWidgetReturnType {
+  activeActions: number;
+  allMembers: UserAvatarsItem[];
+  teamColor: string;
+  currentTokenBalance: BigNumber;
+  nativeToken:
+    | {
+        __typename?: 'Token' | undefined;
+        decimals: number;
+        name: string;
+        symbol: string;
+        type?: TokenType | null | undefined;
+        avatar?: string | null | undefined;
+        thumbnail?: string | null | undefined;
+        tokenAddress: string;
+      }
+    | undefined;
```

Comment:
We can import `Token` from `~ types` and leave it like this `nativeToken: Token | undefined`. This way if the fields of the `Token` model type are updated in the future, we don't have to come back and manually update this type.